PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListViewModel.swift

```diff
@@ -35,10 +35,11 @@ protocol ItemsListViewModel: AnyObject {
     func item(with itemID: ItemIdentifier) -> ItemsListItemPresenter?
     func filterButton(with id: ItemsListFilter) -> TopicChipPresenter
     func selectCell(with: ItemsListCell<ItemIdentifier>)
-    func shareItem(with: ItemsListCell<ItemIdentifier>)
 
-    func toggleFavorite(_ cell: ItemsListCell<ItemIdentifier>)
-    func archive(_ cell: ItemsListCell<ItemIdentifier>)
-    func delete(_ cell: ItemsListCell<ItemIdentifier>)
+    func shareAction(for objectID: ItemIdentifier) -> ReadableAction?
+    func favoriteAction(for objectID: ItemIdentifier) -> ReadableAction?
+    func overflowActions(for objectID: ItemIdentifier) -> [ReadableAction]?
+    func swipeActions(for objectID: ItemIdentifier) -> [UIContextualAction]
```

Comment:
```suggestion
    func trailingSwipeActions(for objectID: ItemIdentifier) -> [UIContextualAction]
```

A small detail, but at least in our current usage, the swipe actions we provide are for the trailing side of the cell. Being a little more explicit in the API might just help in clarity, especially if we may want to add leading actions one day.